### PR TITLE
Feature/yuyu/create tap pay gateway

### PR DESCRIPTION
**Pull request by yuyu-TZ321**
add Tap Pay

### CODE REVIEW

File: src/TapPayPaymentGateway.php [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+
+    /**
+     * @param bool $checkPhoneNumber
+     * @param bool $checkNationalId
+     * @return TapPayPaymentGateway
+     */
+    public function setResultUrl($checkPhoneNumber, $checkNationalId)
+    {
+        $this->resultUrl = [
```
Comment (scott-vt) [inline src/TapPayPaymentGateway.php:64]: ?

File: src/TapPayPaymentGateway.php [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @return TapPayPaymentGateway
+     */
+    public function setCardholderVerify($frontendRedirectUrl, $backendNotifyUrl)
+    {
+        $this->cardholderVerify = [
```
Comment (scott-vt) [inline src/TapPayPaymentGateway.php:79]: ?

File: src/TapPayPaymentGateway.php (hunk 1 above)
Comment (yuyu-TZ321) [reply src/TapPayPaymentGateway.php:64]: done.

File: src/TapPayPaymentGateway.php (hunk 2 above)
Comment (yuyu-TZ321) [reply src/TapPayPaymentGateway.php:79]: done.

**Review by scott-vt — APPROVED** [review]

**Pull request merged**